tools/wptrunner/wptrunner/formatters/chromium.py

```diff
@@ -125,30 +125,34 @@ def _get_expected_status_from_data(self, actual_status, data):
         is expected. So we use the "expected" status if it's there or fall back
```

Comment:
```suggestion
        is expected. So we use the "expected" statuses if it's there or fall back
```

Reply:
`delete "expected" from |data| if it is the same as "status` still correct?

Reply:
..similar to above. And yes, the "delete expected..." is still correct - that code lives in mozlog, it wasn't changed here.